docs/_source/docs/resolvers.rst

```diff
@@ -81,6 +105,49 @@ Refer to `sceptre-resolver-cmd <https://github.com/Sceptre/sceptre-resolver-cmd/
 
 .. _stack_attr_resolver:
 
+select
+~~~~~~
+
+This resolver allows you to select a specific index of a list of items. This is great for combining
+with the ``!split`` resolver to obtain part of a string. This function works almost the same as
+CloudFormation's ``!Select`` intrinsic function, **except you can use this with negative indexes to
+select with a reverse index**.
```

Comment:
What about "except you can use this with negative indices to select from the end of the list." ?